ArduPlane/quadplane.cpp

```diff
@@ -905,6 +912,19 @@ void QuadPlane::hold_hover(float target_climb_rate)
     run_z_controller();
 }
 
+float QuadPlane::get_pilot_throttle()
+{
+    // get normalized throttle [0,1]
+    float throttle_in = (float)plane.channel_throttle->get_control_in() / plane.channel_throttle->get_range();
```

Comment:
Can you add more parentheses (or use a static_cast) here? You're really trusting the precedence order to promote the int to a float, and it'd be easy to break that in the future.